scooby/report.py

```diff
@@ -269,59 +272,43 @@ def colspan(html, txt, ncol, nrow):
             elif nrow % 2 == 0:
                 html += "background-color: #ddd;"
             html += border + " colspan='"
-            html += str(2 * ncol) + "'>%s</td>\n" % txt
+            html += f"{ncol}'>{txt}</td>\n"
```

Comment:
This changes the ncol behaviour. ncol before meant than in a column of ncol there was one column for package names and and one column for version numbers. Now it means that ncol has to account for package names and version numbers separately.